frontend/app/src/components/settings/frontend/AddressNamePrioritySetting.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <div
+      class="text-subtitle-1"
+      v-text="t('eth_address_book.hint.priority.title')"
+    />
+    <settings-option
+      #default="{ error, success, update }"
+      setting="addressNamePriority"
+      @finished="resetCurrentAddressNamePriorities"
+    >
+      <prioritized-list
+        :value="currentAddressNamePriorities"
+        :all-items="availableCurrentAddressNamePriorities()"
+        :item-data-name="
+          t('address_name_priority_setting.data_name').toString()
+        "
+        :disable-add="true"
+        :disable-delete="true"
+        :status="{ error, success }"
+        @input="update"
```

Comment:
A thing that would probably need to be done after changing the priority, is to also re-fetch the names, because, the frontend caches the names and the priority change won't be instantly reflected otherwise.

https://github.com/rotki/rotki/blob/9418c8059c1a162b0b5cd39314e6e6e1e08504d1/frontend/app/src/store/balances/ethereum-names.ts#L82